xml/System.Net/IPAddress.xml

```diff
@@ -1686,10 +1714,12 @@
         <Parameter Name="ipString" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="ipString">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="ipString">A character span that contains an IP address in dotted-quad notation for IPv4 and in colon-hexadecimal notation for IPv6.</param>
+        <summary>Converts an IP address represented as a character span to an IPAddress instance.</summary>
+        <returns>An <see cref="T:System.Net.IPAddress" /> instance.</returns>
```

Comment:
Should we say something like this instead?

```suggestion
        <returns>The converted IP address.</returns>
```

We should avoid simply repeating the info that is auto-generated. (I know the other one was like this as well)